controllers/admin/AdminSeur15Controller.php

```diff
@@ -291,61 +304,66 @@ public function getAllOrders($id = null, $reference = null, $date_start = null,
 		
 		if (trim($country) != '')
 			$where[] = ' a.country like = \'%'.pSQL($country).'%\'   ';
+	
+		if (trim($order_state) != '')
+			$where[] = ' os.id_order_state = '.pSQL($order_state).'   ';
 		
 		if (!empty($where))
 			$sql .= ' WHERE '.implode(' AND ', $where);
 			
 		$sql .= ' ORDER BY o.id_order DESC';
 		return DB::getInstance()->executeS($sql);
-	
 	}
+	
 	public function printLabels($id_orders = array(), $type = 'txt')
 	{
 		try
 		{
-				ob_end_clean();
-				header('Content-type: text/plain');
-				header('Content-Disposition: attachment; filename=seur_labels-'.date('Y-m-d h:i:s', strtotime('now')).'.txt');
-				header('Content-Transfer-Encoding: binary');
-				header('Accept-Ranges: bytes');
-		if (!is_array($id_orders)) $id_orders = (array)$id_orders;
-		$directory = _PS_MODULE_DIR_.'seur/files/deliveries_labels/';
-		if ($type == 'txt')
-		{
+			ob_end_clean();
+			header('Content-type: text/plain');
+			header('Content-Disposition: attachment; filename=seur_labels-'.date('Y-m-d h:i:s', strtotime('now')).'.txt');
+			header('Content-Transfer-Encoding: binary');
+			header('Accept-Ranges: bytes');
```

Comment:
This statement is not valid. Please respect the norm, the assignation has to be on a new blank line.